### PR TITLE
Fix double conversion issues

### LINKED ISSUE #5049: Double conversion at custom asset price if price is denoted in main currency
## Problem Definition

If a user has input a custom asset, say an ETF, and wants to track it in rotki in the same currency as the main currency, say EUR, then a double conversion is happening.

To reproduce:
- Set main currency to EUR (or anything other than USD)
- Create a new custom asset, with a custom price in EUR
- Go back to the dashboard and you will see the price is going through double conversion

For example in this case I put `93.75` but it appears as `93.7500775884375`

2022-10-29-002222_408x143_scrot

## Task

Fix it

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #5049
## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`5080` Temporarily fix the double conversion issue for manual latest price.
```
Comment (LefterisJP) [inline docs/changelog.rst:5]: Please write user facing changelog entries.

1. Temporarly is wrong. The users don't care.
2. What is the double conversion issue? I am a user. I have no idea what you mean.

```suggestion
* :feature:`5080` In the manual balances section, the should no longer be any double conversion. So 1 euro should always be one euro.
```

**Review by LefterisJP — COMMENTED** [review]
It does not seem to work.



The price for this was supposed to be 94.78 (and is also returned as such from the backend) and is even shown as that in the manual balances ... but then when trying to edit it it's wrong.

**Review by LefterisJP — COMMENTED** [review]
Also as I wrote in discord:

```
 {
    "code": "too_big",
    "maximum": 2,
    "inclusive": true,
    "type": "array",
    "path": [
      "assets",
      "eip155:1/erc20:0xec67005c4E498Ec7f55E092bd1d35cbC47C91892"
    ],
    "message": "Array must contain at most 2 element(s)"
  },
```

It seems to expect 2 per entry while now with the boolean it's 3. It's for the `/assets/prices/latest`. You had said we can add an always False boolean there and you can just ignore it. Part of the ugly hack

**Comment by lukicenturi** [comment]
> It does not seem to work.
> 
> 
> 
> The price for this was supposed to be 94.78 (and is also returned as such from the backend) and is even shown as that in the manual balances ... but then when trying to edit it it's wrong.

Can you check again, it should be fixed now

**Comment by lukicenturi** [comment]
> > It does not seem to work.
> > 
> > The price for this was supposed to be 94.78 (and is also returned as such from the backend) and is even shown as that in the manual balances ... but then when trying to edit it it's wrong.
> 
> Can you check again, it should be fixed now

Is it still happens with this PR?

**Comment by lukicenturi** [comment]
Wait, there is a bug, I'm fixing it.

File: frontend/app/src/store/settings/general.ts [hunk 2]
```diff
@@ -62,7 +62,7 @@ export const useGeneralSettingsStore = defineStore('settings/general', () => {
   const floatingPrecision: ComputedRef<number> = computed(() =>
     get(uiFloatingPrecision)
```
Comment (kelsos) [inline frontend/app/src/store/settings/general.ts:63]: This is also a doubly wrapped computed right?

File: frontend/app/src/store/balances/index.ts [hunk 3]
```diff
@@ -28,15 +31,32 @@ export const useBalancesStore = defineStore('balances', () => {
...
+
+        if (price.isCurrentCurrency) {
+          price.usdPrice = price.value.dividedBy(rate);
+        }
+      }
+    }
+
+    updateChainPrices(pricesConvertedToUsd);
+    updateManualPrices(pricesConvertedToUsd);
+    updateExchangePrices(pricesConvertedToUsd);
```
Comment (kelsos) [inline frontend/app/src/store/balances/index.ts:59]: I know this is temporary, but can you add some unit tests for this logic here.

**Review by kelsos — COMMENTED** [review]
@lukicenturi I left some comments, let me know if something is not clear enough

File: frontend/app/src/store/settings/general.ts (hunk 2 above)
Comment (lukicenturi) [reply frontend/app/src/store/settings/general.ts:63]: Oops I missed that

File: frontend/app/src/store/balances/index.ts (hunk 3 above)
Comment (lukicenturi) [reply frontend/app/src/store/balances/index.ts:59]: Unit test is added.

File: docs/changelog.rst [hunk 4]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`5080` In the manual balances section, the should no longer be any double conversion. So 1 euro should always be one euro.
```
Comment (LefterisJP) [inline docs/changelog.rst:5]:
```suggestion
* :feature:`5080` For custom assets with custom price there should no longer be any double conversion. So 1 euro should always be one euro.
```

**Review by LefterisJP — COMMENTED** [review]
Seems to work fine from the test I ran. But ... the frontend e2e tests seem to still fail

**Review by LefterisJP — APPROVED** [review]
LGTM. Let's see during testing how it goes.

**Pull request merged**